docs/ycsb.md

```diff
@@ -28,8 +28,10 @@ spec:
       workloads:
         - workloada
         - workloadb
-      options_load: '-p mongodb.url="mongodb://mongo/ycsb?"' #passed as is to ycsb when loading database
-      options_run: '-p mongodb.url="mongodb://mongo/ycsb?" -threads 10 -target 100'
+      options_load: "-p mongodb.url='mongodb://mongo-0.mongo/ycsb?'" #passed as is to ycsb when loading database
```

Comment:
@aakarshg Any reason for the change from '->" ? It's not an issue just curious why the change.

Reply:
the solution to the issue was to either do this here in the cr, or do that in j2 file, @rsevilla87 suggested changes with cr worked so i tried the same and it worked. but given it looks like changes to j2 file also work, i'll go with that so we cna merge this, and not impact crs.